pkg/meta/parse_test.go

```diff
@@ -367,7 +368,7 @@ func TestParseStorageWithBoltDB(t *testing.T) {
 }
 
 func TestParseStorageDynamoWrapper(t *testing.T) {
-	skipIt(t)
+	tskip.SkipS3(t)
```

Comment:
This should probably be a Dynamo check. Or both? At least Dynamo.

Reply:
just Dynamo, test uses local storage.